frontend/src/components/share/FilePreview.tsx

```diff
@@ -115,22 +116,37 @@ const ImagePreview = () => {
 
 const TextPreview = () => {
   const { shareId, fileId } = React.useContext(FilePreviewContext);
-  const [text, setText] = useState<string | null>(null);
+  const [ text, setText ] = useState<string>("");
 
   useEffect(() => {
     api
       .get(`/shares/${shareId}/files/${fileId}?download=false`)
-      .then((res) => setText(res.data));
-  }, [shareId, fileId]);
+      .then((res) => setText(res.data ?? "Preview couldn't be fetched."));
+  }, [ shareId, fileId ]);
+
+  const options = {
+    overrides: {
+      pre: {
+        props: {
+          style: {
+            backgroundColor: "rgba(50, 50, 50, 0.5)",
```

Comment:
![image](https://github.com/stonith404/pingvin-share/assets/91284394/62d9011e-db37-418d-b04e-0aaa5ef3b3f3)

I think this looks alright, what do you think? `rgba(220, 220, 220, 0.5)`

> const {colorScheme} = useTheme();

it works but I get the error: `Property 'colorScheme' does not exist on type 'Theme'`

Reply:
Yeah, that looks great! 

> it works but I get the error: Property 'colorScheme' does not exist on type 'Theme' (only in the TextPreview, works fine in the global.style.tsx)

My bad, you should use `useMantineTheme` instead of `useTheme`.